unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/ParcelScene.cs

```diff
@@ -544,16 +544,25 @@ public BaseComponent EntityComponentCreateOrUpdate(string entityId, CLASS_ID_COM
             {
                 MessageDecoder.DecodeTransform(data, ref DCLTransform.model);
 
+                TransformLerpController lerpController = null;
                 if (!entity.components.ContainsKey(classId))
+                {
                     entity.components.Add(classId, null);
 
+                    lerpController = entity.gameObject.AddComponent<TransformLerpController>();
```

Comment:
Caution: enabling lerping for all the transforms will cause a lot of unwanted behaviour. We should at least enable this only on transforms that we are landed to. Remember the air balloon is animation based, this could break other system-based movements.

Reply:
Right now the TransformLerpController is doing nothing different than the behaviour we already had, because we are not using its `lerpingSpeed`.

I just left that component there so that when we introduce transform lerping we could just start using that speed property (be it by calculating the speed before or reading it from a property of the SDK component model), and we could also add a different lerping speed for rotation and scaling.

Reply:
I removed the TransformLerpController usage